src/llm/http_llm_calculator.cc

```diff
@@ -580,11 +604,25 @@ class HttpLLMCalculator : public CalculatorBase {
                     } else {  // finish generation
                         OVMS_PROFILE_SCOPE("Generation of last streaming response");
                         std::string response = packIntoServerSideEventMessage(serializeStreamingChunk(this->streamer->end(), finishReason, this->request->getEndpoint()));
-                        response += packIntoServerSideEventMessage("[DONE]");
-                        SPDLOG_LOGGER_DEBUG(llm_calculator_logger, "Generated complete streaming response: {}", response);
-                        // Produce last message, but do not produce loopback packets anymore so this is last Process() call
+
+                        // If usage statistics are not supposed to be included, then this is the last chunk
+                        if (!this->request->getStreamOptions().includeUsage)
```

Comment:
If I read it correctly, we never send [DONE] in case there are usage statistics requested? Is it following the specs? I think vllm benchmarking script waits for that message

Reply:
If `include_usage` is set, then [DONE] is set along with the usage chunk since in that case usage chunk is the last one. 
Otherwise we send [DONE] with last chunk with the content as we've been doing so far.